requirements/pip.txt

```diff
@@ -0,0 +1,11 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    make upgrade
+#
+wheel==0.35.1             # via -r requirements/pip.in
+
+# The following packages are considered to be unsafe in a requirements file:
```

Comment:
Hmm, it looks like `--allow-unsafe` didn't work.  It seems to be working for other people, not sure if we're just using it wrong somehow or it broke recently.  Haven't had time to investigate in depth yet.

Reply:
This comment was added by pip-tools when I ran it without '--allow-unsafe' flag, pip and setuptools were commented out
It has now added the dependencies but didn't remove the comment

Reply:
Ah, missed that the dependencies themselves aren't actually commented out.  Odd, but I guess that works.